### PR TITLE
move deps into gx

### PR DESCRIPTION
**Pull request by whyrusleeping**
This PR has #6 as a prereq to make things easier on me

### CODE REVIEW

**Comment by jbenet** [comment]
@whyrusleeping add a Makefile? we should have a standard:
- makefile for gx-ified repos
- section for the readme

File: README.md [hunk 1]
```diff
@@ -96,3 +96,9 @@ func echoStream(s smux.Stream) {
   fmt.Println("closing stream")
 }
 ```
+
+## Dependencies
+go-stream-muxer uses gx to manage vendored dependencies. While this package may
+work just fine without the vendored deps, to avoid upstream changes breaking
```
Comment (jbenet) [inline README.md]: > while this package may work just fine without the vendored deps

what does this mean? how could it work without the deps?

**Comment by whyrusleeping** [comment]
@jbenet changed the paths to the gx paths

File: multistream/multistream.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ package multistream
 import (
 	"net"
 
-	mss "github.com/whyrusleeping/go-multistream"
+	mss "gx/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```
Comment (jbenet) [inline multistream/multistream.go:8]: @whyrusleeping this is fine with me but is not go-gettable. paths can be:

```
mss "ipfs.io/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```

no?

or similarly:

```
mss "gx.io/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```

Comment (whyrusleeping) [inline multistream/multistream.go:8]: When we get to the point where thats actually possible we can discuss switching to that, but for now its non-trivial to be able to go-get gx packages.

**Pull request merged**